### DISCUSSION TITLE
June 2021 Priorities, Part 2

### DISCUSSION THREAD

**Discussion by jefftangx** [athensresearch/athens#1354] [Goals]
- [Notes](https://roamresearch.com/#/app/athensresearch/page/OfODIDAh1)
- [June 2021 Priorities, Part 2 Here](https://github.com/athensresearch/athens/discussions/1271)

# Frontend + Backend

<details>
<summary>P1: Self-Hosted Collaboration | Status: In Progress | @neotyk @shanberg @tangjeff0 </summary>

- Why: highest, most differentiated product we can sell; improves internal productivity
- [Project Board](https://github.com/athensresearch/athens/projects/16)

</details>

<details>
<summary>P2: DB Switching | Status: In Progress | @shanberg @sid597 </summary>

- Why: improves UX for when we have collaborative knowledge graphs -- will want to be able to switch between local and remote graphs easily.
- #882 

</details> 

# Community

<details>
<summary>P2: ramp up Head of Community | Status: Joel starting full-time 6/18/21 | @joelhans @mateicanavra </summary>

- Why: community is central to Athens differentiation

</details> 

<details>
<summary>P2: Scaling GitHub | Status: in progress | Next steps: create/delete labels | @tangjeff0 </summary>

- Why: we need to master GitHub features to build Athens at scale
- Notes: https://roamresearch.com/#/app/athensresearch/page/kiQNaE1Q5

</details> 

<details>
<summary>P2: Launch Website + Survey | Status: in progress | Next Steps: launch, giveaway for responding? | @tangjeff0 @shanberg @mateicanavra </summary>

- Why: we need a 1) a central entrypoint to Athens that is not GitHub/Twitter/Discord/Handbook, 2) we need to begin marketing Athens to our ideal customer to gain adoption and monetization

</details> 

# Core

<details>
<summary>NYC Trip 6/20/21 - 6/27/21 | </summary>

- Improve morale and teambuilding for core team
- Have a meetup for users and contributors

</details>

**Comment by joelhans** [comment]
It's becoming quite clear that the previous goal of _finishing_ the handbook this month is not going to happen, but I think there are a few places where we can more safely set goals:

- Add issues in handbook repo for every missing doc and doc that needs refinement (aka scope out the entire project)
- Finish user documentation for Athens app
- Finish contributing documentation based on Scaling GitHub notes and various community call+core team conversations
